tests/test_client.py

```diff
@@ -346,7 +348,7 @@ async def test_interceptor(client: temporalio.client.Client, worker: Worker):
     await handle.result()
     await handle.cancel()
     # Ignore this error
-    with pytest.raises(temporalio.client.RPCError):
```

Comment:
Hehe I am noticing someone is coming around to more frequent relative imports

![](https://wompampsupport.azureedge.net/fetchimage?siteId=7575&v=2&jpgQuality=100&width=700&url=https%3A%2F%2Fi.kym-cdn.com%2Fphotos%2Fimages%2Fnewsfeed%2F000%2F804%2F849%2F54c.jpg)

Reply:
Ha, I am actually deviating from the suggestions of the community. In fact, I had to update the README to basically say "We know the Google Style Guide says not to use relative imports, but we are anyways in tests but _not library code_!"